<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportScenarioDialog</class>
 <widget class="QDialog" name="ExportScenarioDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>735</width>
    <height>575</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Scenario</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QSplitter" name="main_splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <layout class="QVBoxLayout" name="level_list_vbox">
       <item>
        <widget class="QLabel" name="scenario_level_list_label">
         <property name="text">
          <string>Scenario levels:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="scenario_level_list"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="preview_tabs_vbox">
       <item>
        <layout class="QHBoxLayout" name="preview_label_hbox">
         <item>
          <widget class="QLabel" name="script_preview_label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Level script preview:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="script_name_label">
           <property name="text">
            <string>N/A</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="script_preview">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialog_button_box">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
